app/AccountancyModule/PaymentModule/presenters/AddCampParticipantsPresenter.php

```diff
@@ -32,36 +35,41 @@ public function __construct(PaymentService $model, IMassAddFormFactory $formFact
 
     public function actionDefault(int $id) : void
     {
-        $this->id = $id;
-
         $group = $this->model->getGroup($id);
 
         if ($group === null || ! $this->isEditable) {
             $this->flashMessage('Neoprávněný přístup ke skupině.', 'danger');
             $this->redirect('GroupList:');
+
+            return;
```

Comment:
je potřeba za redirectem přidavat return?

Reply:
Důvod je především ten, že IDE a PHPStan nevěděli, že redirect vždy vyhazuje výjimku, takže nefungovalo:

```
$group = something();

if ($group === null) {
    $this->redirect('Somewhere');
}

$group->doSomething(); // tohle je bezpečné, protože `$group` nikdy nemůže být null
```

Každopádně už mi ani PHPStan ani IDE nic nehlásí (PHPStorm dokonce správně hlásí, že je to nedosažitelný statement).

Reply:
Odebral jsem